internal/controller/datadogagent/feature/admissioncontroller/feature.go

```diff
@@ -28,21 +28,31 @@ func init() {
 }
 
 type admissionControllerFeature struct {
-	mutateUnlabelled       bool
-	serviceName            string
-	webhookName            string
-	agentCommunicationMode string
-	agentSidecarConfig     *AgentSidecarInjectionConfig
-	localServiceName       string
-	failurePolicy          string
-	registry               string
-	serviceAccountName     string
-	owner                  metav1.Object
+	validationWebhookConfig *ValidationConfig
+	mutationWebhookConfig   *MutationConfig
```

Comment:
are more configuration options definitely planned? could we just use `*bool`?

Reply:
We are planning on having more configuration, for example a `unlabeled` configuration per webhook type.